_includes/button.html

```diff
@@ -7,13 +7,13 @@
       class="button"
       href="{{ button.link | replace: "$LINK", button.link | relative_url }}"
       {% if button.tooltip %}
-        data-tooltip="{{ button.tooltip }}"
+        data-tooltip="{{ button.tooltip | xml_escape }}"
```

Comment:
Add this in a bunch of places to prevent HTML parsing errors, e.g. `button.tooltip` having a value of `"Effective" methods of communication` (where the quotes screw up `data-tooltip=""`).

Reply:
In the case of `data-tooltip`, Tippy.js will handle the HTML entities yes. But in other cases there are better escape methods to use. I've created this issue to follow up on this: https://github.com/greenelab/lab-website-template/issues/288